apps/web/src/utils/trpc/types.ts

```diff
@@ -5,7 +5,7 @@ export type GetPollApiResponse = {
   title: string;
   location: string | null;
   description: string | null;
-  options: { id: string; start: Date; duration: number }[];
+  options: { id: string; start: Date; startTime: Date; duration: number }[];
```

Comment:
Can you suggest a better name?

Reply:
I'm with you but to avoid needing to rename multiple times I would prefer to keep it as is. It will make sense once we drop `start` and only `startTime` will remain.